gobblin-core/src/test/java/org/apache/gobblin/publisher/BaseDataPublisherTest.java

```diff
@@ -588,4 +621,23 @@ private State buildDefaultState(int numBranches)
 
     return state;
   }
+
+  private WorkUnitState buildTaskState(int numBranches) {
+    WorkUnitState state = new WorkUnitState();
+
+    state.setProp(ConfigurationKeys.EXTRACT_NAMESPACE_NAME_KEY, "namespace");
+    state.setProp(ConfigurationKeys.EXTRACT_TABLE_NAME_KEY, "table");
+    state.setProp(ConfigurationKeys.WRITER_FILE_PATH_TYPE, "namespace_table");
+    state.setProp(ConfigurationKeys.FORK_BRANCHES_KEY, numBranches);
+    state.setProp(ConfigurationKeys.DATA_PUBLISHER_FINAL_DIR, "/data/output");
+    state.setProp(ConfigurationKeys.WRITER_OUTPUT_DIR, "/data/working");
+    if (numBranches > 1) {
+      for (int i = 0; i < numBranches; i++) {
+        state.setProp(ConfigurationKeys.DATA_PUBLISHER_FINAL_DIR + "." + i, "/data/output" + "/branch" + i);
+        state.setProp(ConfigurationKeys.WRITER_OUTPUT_DIR+ "." + i, "/data/working" + "/branch" + i);
```

Comment:
nit: space between WRITER_OUTPUT_DIR and  "."

Reply:
+1